src/App.tsx

```diff
@@ -37,59 +38,126 @@ export function getReorderedGoods(
   // Sort and reverse goods if needed
   // eslint-disable-next-line no-console
   console.log(sortType, isReversed);
+  visibleGoods.sort((prevGood, nextGood) => {
+    switch (sortType) {
+      case SortType.ALPHABET:
+        return prevGood.localeCompare(nextGood);
+
+      case SortType.LENGTH:
+        return prevGood.length - nextGood.length;
+
+      default:
+        return 0;
+    }
+  });
```

Comment:
Don't start sort if you haven't sort type + don't check sort type inside iterator, check it before
```suggestion
  switch (sortType) {
    case SortType.ALPHABET: 
      visibleGoods.sort((firstGood, secondGood) => (
        firstGood.localeCompare(secondGood);
      )
    
    case .....
```